Assignment3A/StringManipulation.java

```diff
@@ -0,0 +1,185 @@
+package Assignment3A;
+
+/**
+ * 
+ * this class contain methods to manipulate strings we can perform
+ * comparison,reverse of string, upper case to lower case etc
+ *
+ */
+public class StringManipulation {
+
+	/**
+	 * To compare two strings
+	 * @param firstString contain a not null case sensitive String
+	 * @param secondString contain a not null case sensitive String
+	 * @return Integer value 1 if strings are equal and 0 if not
+	 */
+	public int compareStrings(String firstString, String secondString) {
+
+		char firstStringCharArray[] = firstString.toCharArray();
+		int firstStringLength = firstString.length();
+
+		char secondStringCharArray[] = secondString.toCharArray();
+		int secondStringLength = secondString.length();
+
+		if (firstStringLength != secondStringLength) {
+			return 0;
+		} else {
+			int charIndex = 0;
+			while (charIndex != firstStringLength - 1) {
+				if (firstStringCharArray[charIndex] != secondStringCharArray[charIndex]) {
+					return 0;
+				}
+				charIndex++;
+			}
+		}
+		return 1;
+	}
+
+	/**
+	 * To get reverse of a string
+	 * @param firstString contain a not null case sensitive String
+	 * @return reverse of string
+	 */
+	public String reverseOfString(String firstString) {

```

Comment:
Parameter name should be relative.You should change "firstString" to more relative name.